Documentation/templating/has_parts.md

```diff
@@ -45,6 +41,71 @@ When using this, you simply use the slot names as tags:
 
 The elements based on their tag names will then be mapped across and appear within the correct slot.
 
+```html
+<div class="my-component">
+  <div class="banner"></div>
+  <div class="items">
+    <li>First item</li>
+    <li>Second item</li>
+  </div>
+</div>
+```
+
+### Overrides
+
+As you can see in the above example, the output will create an invlid HTML when we know that the standards says that an `<li>` element is requiring to live within an `<ul>`element. This can we solve by overriding the tag type.
```

Comment:
```suggestion
As you can see in the above example, the output will create an invalid HTML when we know that the standards says that an `<li>` element is requiring to live within an `<ul>`element. This can we solve by overriding the tag type.
```